WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewView.swift

```diff
@@ -0,0 +1,252 @@
+                                               value: "Regular price: $15",
+                                               image: UIImage.priceImage,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.inventory,
+                                               value: Localization.inStock,
+                                               image: UIImage.inventoryImage,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.categories,
+                                               value: "Food, snack, sweet",
+                                               image: UIImage.categoriesIcon,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.tags,
+                                               value: "yummy, candy, chocolate",
+                                               image: UIImage.tagsIcon,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.shipping,
+                                               value: "Weight: 1kg\nDimension: 15 x 10 x 3 cm",
+                                               image: UIImage.shippingImage,
+                                               isLoading: viewModel.isGeneratingDetails)
+                    }
+                    .cornerRadius(Layout.cornerRadius)
+                }
+
+                // Feedback banner
+                FeedbackView(title: Localization.feedbackQuestion,
+                             backgroundColor: .init(uiColor: .init(light: .withColorStudio(.wooCommercePurple, shade: .shade0),
+                                                                   dark: .tertiarySystemBackground)),
+                             onVote: { vote in
+                    viewModel.handleFeedback(vote)
+                })
+                .renderedIf(viewModel.isGeneratingDetails == false)
+            }
+            .padding(insets: Layout.insets)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    if viewModel.isGeneratingDetails {
+                        ActivityIndicator(isAnimating: .constant(true), style: .medium)
+                    } else {
+                        Button("Save as draft") {
```

Comment:
Super nit: I think we can localize this string.

Reply:
Updated in a70e2e2ccc0f45410b9c1df3e0e67ab864968e94.